front_end_pocket_poll/lib/events_widgets/events_list.dart

```diff
@@ -1,37 +1,112 @@
 import 'package:flutter/material.dart';
 import 'package:front_end_pocket_poll/imports/events_manager.dart';
 import 'package:front_end_pocket_poll/imports/globals.dart';
-import 'package:front_end_pocket_poll/imports/groups_manager.dart';
 import 'package:front_end_pocket_poll/models/event_card_interface.dart';
-import 'package:front_end_pocket_poll/models/group.dart';
 
 class EventsList extends StatefulWidget {
+  static final int eventsTypeNew = 0;
+  static final int eventsTypeClosed = 1;
+  static final int eventsTypeConsider = 2;
+  static final int eventsTypeVoting = 3;
+  static final int eventsTypeOccurring = 4;
+
   final List<EventCardInterface> events;
+  final int eventsType;
```

Comment:
This is a really small change, but personally I think it would make it easier to read. I think the variable should be called "eventType".

Since there's only one of these types (consider,voting,etc)